public/CSS Files/style2.css

```diff
@@ -0,0 +1,250 @@
+@import url('https://fonts.googleapis.com/css?family=Oxanium|Trade+Winds&display=swap');
+
+.grid-container {
+    display: grid;
+    width: 850px;
+    height: 700px;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 2fr 8fr 2fr;
+    grid-gap: 1rem;
+    grid-template-areas:
+        "header  map map"
+        "sideBar  map  map"
+        "footer  footer    footer";
+}
+
+
+body{
+    background:   lightgrey;
+    display: flex;
+    justify-content: center;
+
+}
+
+
+/*---------Header---------*/
+
+
+.header {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-weight: 900;
+    height: 10vw;
+    width: 80vw;
+    border-radius: 25px;
+    box-shadow: 0px 12px 15px -10px black;
+    background: blue;
+    background: linear-gradient(blue, grey);
+}
+
+#header {
+    width: auto;
+    height: auto;
+}
+
+#head-content{
+
+    text-align: center;
+    font-family:Trade Winds ;
+    font-size: 5rem;
+    color: white;
+    position: relative;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    text-decoration : none;
+}
+#head-link{
+    text-decoration: none;
+    color: white;
+}
+
+/*---------side-bar---------*/
+
+
+.sideBar {
+    grid-area: sideBar;
+    border-radius: 25px;
+    background: linear-gradient(grey, blue);
+    width: 20vw;
+    height: 60vh;
+    margin-top: 10px;
+    margin-right: 5px;
+    color:white;
+}
+#name{
+    display: flex;
+    justify-content: flex-start;
+    margin-top: 10px;
+    font-size: xx-large;
```

Comment:
I think this is the rule causing the big text in the name input on the restaurant page